kubeflow/common/istio/kustomization.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-- ./upstream/istio-1-9-0/kubeflow-istio-resources/base # {"$kpt-set":"istio-istio-base"}
```

Comment:
It looks like `istio-1-9-0/` is not created when pulling from upstream: `bash ./pull-upstream.sh`